script/core/interaction/CreateProfile.sol

```diff
@@ -13,7 +13,7 @@ contract CreateProfile is Script {
     string public name = "";
     Metadata public metadata = Metadata({protocol: uint256(0), pointer: ""});
     address public owner = address(0);
-    address[] public members = [];
+    address[] public members;
```

Comment:
This should be filled by the user running the script right? is this mentioned in the readme?

Reply:
Yes https://github.com/allo-protocol/allo-v2.1/blob/chore/fix-default-script/script/core/interaction/README.md